backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc

```diff
@@ -1,25 +1,32 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="oadp-backing-up-applications-restic-doc"]
-= Backing up applications with Restic
+= Backing up applications with File System Backup: Kopia or Restic
 include::_attributes/common-attributes.adoc[]
 :context: backing-up-applications
 
 toc::[]
 
-If your cloud provider does not support snapshots or if your applications are on NFS data volumes, you can create backups by using Restic.
+OADP enables you to back up and restore Kubernetes volumes attached to pods from the file system of the volumes, called File System Backup (FSB) or Pod Volume Backup. The data movement is accomplished by using modules from open source backup tools, Restic and Kopia.
+
+If your cloud provider does not support snapshots, or if your applications are on NFS data volumes, you can create backups by using FSB.
 
 [NOTE]
 ====
 link:https://restic.net/[Restic] is installed by the OADP Operator by default.
```

Comment:
I think we can mention here 

Restic / Kopia,  instead of only restic, and add this kopia link as well: https://kopia.io/